sys/include/usb/usbus.h

```diff
@@ -176,6 +177,7 @@ typedef enum {
     USBUS_EVENT_USB_SOF,        /**< USB start of frame received    */
     USBUS_EVENT_USB_SUSPEND,    /**< USB suspend condition detected */
     USBUS_EVENT_USB_RESUME,     /**< USB resume condition detected  */
+    USBUS_EVENT_USB_ACTIVE,     /**< USB configuration set and EPs activated */
```

Comment:
Initially, I was thinking about `USBUS_EVENT_USB_EP_ACTIVATED` but the name is pretty long. The question is what it should signal? Should it signal that USB configuration finished or whether the EPs are active. In case of MSC, it doesn't matter.

Reply:
Does an interface really needs to know when the configuration is finished ? I am not sure, so maybe signalling EPs need to be active would be better there.